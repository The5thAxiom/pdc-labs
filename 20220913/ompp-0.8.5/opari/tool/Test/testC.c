int main() {

  #pragma omp flush

  #pragma omp flush (i, j, k)

}
