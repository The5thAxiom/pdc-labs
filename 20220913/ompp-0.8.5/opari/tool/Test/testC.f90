      program testC

      !$OMP FLUSH

      !$OMP FLUSH (I, J, K)

      end
