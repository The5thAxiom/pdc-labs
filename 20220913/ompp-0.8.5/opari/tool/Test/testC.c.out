#ifdef _POMP
#  undef _POMP
#endif
#define _POMP 200110

#include "testC.c.opari.inc"
#line 1 "testC.c"
int main() {

POMP_Flush_enter(&omp_rd_38);
#line 3 "testC.c"
  #pragma omp flush
POMP_Flush_exit(&omp_rd_38);
#line 4 "testC.c"

POMP_Flush_enter(&omp_rd_39);
#line 5 "testC.c"
  #pragma omp flush (i, j, k)
POMP_Flush_exit(&omp_rd_39);
#line 6 "testC.c"

}
